modules/nf-core/tcoffee/consensus/main.nf

```diff
@@ -0,0 +1,62 @@
+process TCOFFEE_CONSENSUS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/t-coffee_pigz:f47b85d70360f1a0':
+        'community.wave.seqera.io/library/t-coffee_pigz:6c9b2f8b97ee55e5' }"
+
+
+    input:
+    tuple val(meta), path(aln)
+    tuple val(meta2), path(tree)
+    val(compress)
+
+    output:
+    tuple val(meta), path("*.{aln,aln.gz}"), emit: alignment
+    path "versions.yml"           , emit: versions
```

Comment:
Some people say we should be ["harshil conform"](https://nf-co.re/docs/contributing/code_editors_and_styling/harshil_alignment).

Reply:
Ahahah nice! Fixed it